### ISSUE TITLE
Vertx 4 support.

### ISSUE THREAD

**Issue by aomsweet** [quarkusio/quarkus#13768]
### Description

Vertx 4.0.0 is released. Please support. Thanks ☕

**Comment by cescoffier** [comment]
The integration of Vert.x 4 is on the roadmap and we already starting working on it.
That integration is going to take time. Many projects need to switch to Vert.x 4 before we can integrate it in Quarkus (Mutiny API of Vert.x (WIP), Hibernate Reactive (WIP), RESTEasy Reactive, Reactive Messaging, Quarkus HTTP...)). 

Once all these boxes get checked, there will still need to do some work in Quarkus as Vert.x 4 contains several breaking changes (which make sense as it's a new major version). Typically the gRPC, security layer, and HTTP routing layer need to be slightly updated. 

I cannot give you an exact date, but we hope to be able to provide a Vert.x 4 based Quarkus in March or April.

**Comment by gastaldi** [comment]
Duplicate of https://github.com/quarkusio/quarkus/issues/9262. Closing
